### PR TITLE
More rebasing stuff

### PR DESCRIPTION
**Pull request by msisov**
Please review

### CODE REVIEW

File: runtime/browser/runtime_geolocation_permission_context.cc [hunk 1]
```diff
@@ -127,7 +127,7 @@ void RuntimeGeolocationPermissionContext::OnPermissionRequestFinished(
     base::Callback<void(bool)> result_callback, bool success) {
   if (success) {
     device::GeolocationProvider::GetInstance()
-        ->UserDidOptIntoLocationServices();
+        ->UserDidOptIntoLocationServices();
```
Comment (rakuco) [inline runtime/browser/runtime_geolocation_permission_context.cc]: It looks like you've just added some extra empty space at the end of the line.

File: runtime/browser/ssl_error_page.h [hunk 2]
```diff
@@ -8,6 +8,8 @@
 #include <string>
 
 #include "base/callback.h"
+
```
Comment (rakuco) [inline runtime/browser/ssl_error_page.h]: Extra empty line.

File: runtime/browser/xwalk_autofill_client.cc [hunk 3]
```diff
@@ -137,6 +137,14 @@ bool XWalkAutofillClient::IsContextSecure(const GURL& form_origin) {
       ssl_status.content_status == content::SSLStatus::NORMAL_CONTENT;
 }
 
+bool XWalkAutofillClient::ShouldShowSigninPromo() {
+    NOTIMPLEMENTED();
+    return true;
+}
+void XWalkAutofillClient::StartSigninFlow() {
```
Comment (rakuco) [inline runtime/browser/xwalk_autofill_client.cc]: There should be an empty line before this method.

File: runtime/browser/xwalk_autofill_client.cc [hunk 4]
```diff
@@ -137,6 +137,14 @@ bool XWalkAutofillClient::IsContextSecure(const GURL& form_origin) {
       ssl_status.content_status == content::SSLStatus::NORMAL_CONTENT;
 }
 
+bool XWalkAutofillClient::ShouldShowSigninPromo() {
+    NOTIMPLEMENTED();
+    return true;
```
Comment (rakuco) [inline runtime/browser/xwalk_autofill_client.cc]: This should probably return `false`, not `true`, as we have no implementation.

File: runtime/browser/xwalk_autofill_client.h [hunk 5]
```diff
@@ -95,8 +97,12 @@ class XWalkAutofillClient : public autofill::AutofillClient {
       const base::string16& profile_full_name) override;
   void OnFirstUserGestureObserved() override;
   bool IsContextSecure(const GURL& form_origin) override;
+  bool ShouldShowSigninPromo() override;
+  void StartSigninFlow() override;
   void SuggestionSelected(int position);
 
+
```
Comment (rakuco) [inline runtime/browser/xwalk_autofill_client.h]: Unneeded.

File: runtime/browser/xwalk_content_browser_client.cc [hunk 6]
```diff
@@ -93,22 +92,6 @@ namespace {
 // The application-wide singleton of ContentBrowserClient impl.
 XWalkContentBrowserClient* g_browser_client = nullptr;
 
-// A provider of services for Geolocation.
```
Comment (rakuco) [inline runtime/browser/xwalk_content_browser_client.cc]: Are you sure about this? It looks like you're removing the code without providing any replacement.

Comment (msisov) [inline runtime/browser/xwalk_content_browser_client.cc]: Fixed.

**Pull request merged**